client/performance-profiler/utils/profiler-version.ts

```diff
@@ -0,0 +1,2 @@
+export const profilerVersion = () =>
+	window.location.pathname.includes( '/sites/performance/' ) ? 'logged-in' : 'logged-out';
```

Comment:
If we ever changed the the page URLs and forgot to update this function we wouldn't want to inadvertently record data that looks correct but is actually bogus. It ruin all the stats up till the point we discovered the issue. Same thing for if we ever added this component to a whole other page.

I think it's better not to assume that there's only two possible profiler versions.

```suggestion
	if ( window.location.pathname.includes( '/sites/performance/' ) ) {
		return 'logged-in';
	} else if ( window.location.pathname.includes( '/speed-test-tool' ) ) {
		return 'logged-out';
	}
	return 'unknown';
```

Reply:
That is true, best to be exhaustive with the expected options here.